shelter/remoteattestation/securitychannel.go

```diff
@@ -46,32 +54,45 @@ func RemoteTlsSetupTCP(address string, mrenclave unsafe.Pointer, mrsigner unsafe
 	if err != nil {
 		return err
 	}
-	C.ra_tls_echo(C.int(sockfd.Fd()), (*C.uchar)(mrenclave), (*C.uchar)(mrsigner))
-	return nil
+        logLevel := C.ENCLAVE_TLS_LOG_LEVEL_DEBUG
+        if strings.EqualFold(logLevelInit, "debug"){
+                logLevel = C.ENCLAVE_TLS_LOG_LEVEL_DEBUG
+        } else if strings.EqualFold(logLevelInit, "info") {
+                logLevel = C.ENCLAVE_TLS_LOG_LEVEL_INFO
+        }
+        C.ra_tls_echo(C.int(sockfd.Fd()), C.enclave_tls_log_level_t(logLevel), C.CString(attester), C.CString(verifier), C.CString(tls), C.CString(crypto))
```

Comment:
Enclave TLS has 6 levels log output. `Please refer to the latest samples/encalve-tls-server/server.c.` for .the detailed infomation.

Reply:
ok